### PR TITLE
added git commands and coding guidelines to readme

### PR DESCRIPTION
**Pull request by gsvamsiBZ**
#44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +19,80 @@ git pull upstream main
...
 15) create component folder in frontend/src and create homepage.js in component folder.
 16) Add proxy to frontend/package.json
+------------------------------------------------
+
+## Git Commands
+### Getting & Creating Projects
+| Command | Description |
+| ------- | ----------- |
+| `git init` | Initialize a local Git repository |
+| `git clone ssh://git@github.com/[username]/[repository-name].git` | Create a local copy of a remote repository |
```
Comment (Karthik-184) [inline README.md:29]: mention who should use ssh and who should use http while cloning through comment

Comment (Karthik-184) [inline README.md:29]: Mention who should use ssh and who should use http while cloning through comment

File: README.md [hunk 2]
```diff
@@ -28,6 +19,80 @@ git pull upstream main
 14) remove and add our code in frontend/src/app.js
 15) create component folder in frontend/src and create homepage.js in component folder.
 16) Add proxy to frontend/package.json
+------------------------------------------------
+
+## Git Commands
+### Getting & Creating Projects
```
Comment (Karthik-184) [inline README.md:25]: Add an empty line after the heading as you have done below

File: README.md [hunk 3]
```diff
@@ -28,6 +19,80 @@ git pull upstream main
...
+### Inspection & Comparison
+
+| Command | Description |
+| ------- | ----------- |
+| `git log` | View changes |
+| `git log --summary` | View changes (detailed) |
+| `git log --oneline` | View changes (briefly) |
+| `git diff [source branch] [target branch]` | Preview changes before merging |
 
+## Coding Guidelines
```
Comment (Karthik-184) [inline README.md:83]: Add an empty line below

File: README.md [hunk 4]
```diff
@@ -28,6 +20,96 @@ git pull upstream main
 14) remove and add our code in frontend/src/app.js
 15) create component folder in frontend/src and create homepage.js in component folder.
 16) Add proxy to frontend/package.json
+------------------------------------------------
+## Steps to set up SSH based authentication:
+- Create a ssh key on your system’s terminal using following command
+    - ssh-keygen -t ed25519 -C "your email id"
+- Press enter for all inputs and it will create two files in ~/.ssh folder - id_ed25519 and id_ed25519.pub
+- Copy the contents of id_ed25519.pub file.
+- In your [Github Account settings ](https://github.com/settings/profile), click on “SSH and GPG Keys”. (Make sure you are logged in using beingzero id).
```
Comment (Siddhartha-bethi) [inline README.md:29]: Remove being zero id

File: README.md [hunk 5]
```diff
@@ -28,6 +20,96 @@ git pull upstream main
...
+| `git checkout -b [branch name] origin/[branch name]` | Clone a remote branch and switch to it |
+| `git branch -m [old branch name] [new branch name]` | Rename a local branch |
+| `git checkout [branch name]` | Switch to a branch |
+| `git checkout -` | Switch to the branch last checked out |
+| `git checkout -- [file-name.txt]` | Discard changes to a file |
+| `git merge [branch name]` | Merge a branch into the active branch |
+| `git merge [source branch] [target branch]` | Merge a branch into a target branch |
+| `git stash` | Stash changes in a dirty working directory |
+| `git stash clear` | Remove all stashed entries |
+
```
Comment (Siddhartha-bethi) [inline README.md:78]: Add git stash pop

**Review by Siddhartha-bethi — CHANGES_REQUESTED** [review]

**Review by Siddhartha-bethi — APPROVED** [review]

**Review by Karthik-184 — APPROVED** [review]

**Pull request merged**